src/conversion/model-to-view-converters.js

```diff
@@ -475,9 +475,9 @@ export function convertElementsInsideMarker( selectionDescriptor ) {
 
 		const viewElement = conversionApi.mapper.toViewElement( modelItem );
 		const addMarker = evt.name.split( ':' )[ 0 ] == 'addMarker';
-		const selectionHandlingMethod = addMarker ? 'setVirtualSelection' : 'removeVirtualSelection';
+		const highlightHandlingMethod = addMarker ? 'setHighlight' : 'removeHighlight';
```

Comment:
I'm not sure about `setHighlight`. First sounds strange that it's `setHighlight` instead of just `highlight`. Second, if we have `remove` the better pair might be `add`, especially when you can add multiple highlights. So I would go with `addHighlight` instead of `setHighlight`.